packages/grid/x-data-grid/tsconfig.json

```diff
@@ -4,5 +4,5 @@
     "noEmit": true,
     "types": ["mocha", "node", "chai", "chai-dom"]
   },
-  "include": ["src"]
+  "include": ["src", "../../../node_modules/@mui/monorepo/test/utils/initMatchers.ts"]
```

Comment:
Is there something preventing us from adding it to the root `tsconfig.json` file of the repo ?

Reply:
Yes, `include` in the root `tsconfig.json` gets overwritten by the package `tsconfig.json` when extending. https://www.typescriptlang.org/tsconfig#extends